packages/three-vrm-core/src/humanoid/VRMHumanoidRig.ts

```diff
@@ -54,15 +57,13 @@ export class VRMHumanoidRig extends VRMRig {
 
         // see the nearest parent position
         let currentBoneName: VRMHumanBoneName | null = boneName;
-        let parentWorldPosition: THREE.Vector3 | undefined;
-        let parentWorldRotation: THREE.Quaternion | undefined;
-        while (parentWorldPosition == null) {
+        let parentBoneWorldPosition: THREE.Vector3 | undefined;
+        while (parentBoneWorldPosition == null) {
           currentBoneName = VRMHumanBoneParentMap[currentBoneName];
           if (currentBoneName == null) {
             break;
           }
-          parentWorldPosition = boneWorldPositions[currentBoneName];
-          parentWorldRotation = boneWorldRotations[currentBoneName];
```

Comment:
[memo]: VRMHumanBoneParentMap was used to reference a rotation of the parent bone, which was problematic when the parent was a non-humanoid bone with rotation.